hazelcast/src/main/java/com/hazelcast/internal/config/LicenseKey.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.config;
+
+public class LicenseKey {
+    private final String licenseKey;
+
+    public LicenseKey(String licenseKey) {
+        this.licenseKey = licenseKey;
+    }
+
+    public String getLicenseKey() {
+        return licenseKey;
+    }
+}
```

Comment:
maybe equals / hashcode / toString (without the actual key)?

Reply:
https://github.com/hazelcast/hazelcast/pull/20446/commits/66601424b00ed564636c7a9733a799e8099c453b